src/nwb2bids/bids_models/_electrodes.py

```diff
@@ -70,12 +70,7 @@ def from_nwbfiles(cls, nwbfiles: list[pydantic.InstanceOf[pynwb.NWBFile]]) -> ty
             raise NotImplementedError(message)
         nwbfile = nwbfiles[0]
 
-        electrical_series = [
-            neurodata_object
-            for neurodata_object in nwbfile.objects.values()
-            if isinstance(neurodata_object, pynwb.ecephys.ElectricalSeries)
-        ]
-        if any(electrical_series) is False:
```

Comment:
Not sure why I originally 

Probably out of simplicity, I suppose

It is quite possible for a file to only contain metadata and no data

Of course, the multiple NWB file case (especially with overlapping metadata) remains a future complication